pkg/operator/controllers/muo/staticresources/config.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: managed-upgrade-operator-config
+  namespace:  openshift-managed-upgrade-operator
+data:
+  config.yaml: |
+    configManager:
+      source: LOCAL
+      localConfigName: managed-upgrade-config
+      watchInterval: 1
+    maintenance:
+      controlPlaneTime: 90
+      ignoredAlerts:
+        controlPlaneCriticals:
+        - ClusterOperatorDown
+        - ClusterOperatorDegraded
+    upgradeWindow:
+      delayTrigger: 30
+      timeOut: 120
+    nodeDrain:
+      timeOut: 45
```

Comment:
does it make sense to have the timeout greater than 1 hour to allow node controller time for force evict before we fail here?

Reply:
MUO has this functionality, so it will do it after the 45 minutes. Whether we want it at 45 is maybe a different question?

Reply:
For the node drain, does it only apply during cluster upgrades, or during any MCP update?  Wondering when/if we can remove the `node_controller`